core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/workspace/config/MachineConfig.java

```diff
@@ -37,6 +37,21 @@
    */
   String MEMORY_REQUEST_ATTRIBUTE = "memoryRequestBytes";
 
+  /**
+   * Name of the attribute from {@link #getAttributes()} which, if present, defines the entrypoint
+   * command to be executed in the machine/container.
```

Comment:
Please add info about its format: Array of strings. An example would be useful as well, `["sh","-c"]`